storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -657,4 +658,16 @@ Options
 
    This option prints |xtrabackup| version and exits.
 
+.. option:: --xtrabackup-plugin-dir=DIRNAME
+
+   The absolute path of the directory that contains the
+   ``keyring_file`` plugin (:file:`keyring_file.so`).
```

Comment:
maybe simply keyring plugin, or keyring plugins?

Reply:
Ready for review